serde/pom.xml

```diff
@@ -81,7 +81,7 @@
       <version>${hppc.version}</version>
     </dependency>
     <dependency>
-      <groupId>com.github.davidmoten</groupId>
+      <groupId>com.google.flatbuffers</groupId>
```

Comment:
does this change introduce any incompatibilities?

Reply:
I don't think so (see warning message in jira ticket), previously for this module we have 2 dependencies from different groups and shade plugin may select one of them randomly. My change makes clear to use only official artefacts from one source.

Reply:
@ayushtkn anything more I can help with review?

Reply:
Should be fine then. Sorry I forgot about this.